bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -272,27 +267,21 @@ private boolean isNgdmSupportAvailable() {
         return nextGenDynamicMediaConfig != null && nextGenDynamicMediaConfig.enabled() &&
             StringUtils.isNotBlank(nextGenDynamicMediaConfig.getRepositoryId());
     }
-
     private void initNextGenerationDynamicMedia() {
         initResource();
         properties = resource.getValueMap();
         String fileReference = properties.get("fileReference", String.class);
+        String smartCrop = properties.get("smartCrop", String.class);
```

Comment:
I can see a String[]  type for smartCrop JCR property in crx/de, could you make sure to be simply String if you don't need String[]?

Reply:
fixed.